### DISCUSSION TITLE
Converge issue

### DISCUSSION THREAD

**Discussion by Yulongggggg** [idaholab/moose#28641] [Q&A Modules: General] [answered]
https://github.com/Yulongggggg/2D_groundstress This is my code link, I am trying to model the situation of biaxial loading of soil. But the code seems unable to converge, could you please tell me my mistakes. Thank you so much!

**Comment by GiudGiud** [comment, marked as answer]
Hello
there are some troubleshooting instructions here
https://mooseframework.inl.gov/moose/application_usage/failed_solves.html
and some advice on preconditioning for porous media / geomecha here
https://mooseframework.inl.gov/modules/porous_flow/solvers.html
